src/views/count.vue

```diff
@@ -97,10 +105,44 @@
     },
     computed: {
       ...mapGetters({
-        product: "product/getCurrent"
+        product: "product/getCurrent",
+        facility: 'user/getCurrentFacility',
       })
     },
+    data(){
+      return{
+        facilityLocation: [] as any,
+        pickerOptions: {} as any,
+        locationId: ""
+      }
+    },
+    async mounted(){
+      this.facilityLocation = await this.store.dispatch("user/getFacilityLocation", {facilityId: this.facility.facilityId, productId: this.product.productId});
```

Comment:
I think you should call service directly, we are not mutating the state, action is not needed

Reply:
Also, improve variable name, I think we are getting `facilityLocations`

Reply:
Removed action from user module and written a method in **vue** file to fetch locations.